test/r2.md

```diff
@@ -2,17 +2,20 @@
 
 **Table of Contents**
 
-- [EOS-15766 Test Resume update process after reboot](#eos-15766-test-resume-update-process-after-reboot)
-- [EOS-15751 Test - Setup R2 upgrade release bundle](#eos-15751-setup-r2-upgrade-release-bundle)
-- [EOS-16927 New API for upgrade: adjust rollback logic for R2 (singlenode)](#eos-16927-new-api-for-upgrade-adjust-rollback-logic-for-r2-singlenode)
-- [EOS-18337: Make bootstrap API salt states for cortx repositories available in deployed environment](#eos-18337-make-bootstrap-api-salt-states-for-cortx-repositories-available-in-deployed-environment)
-- [EOS-17831: Bootstrap Storage-Set-ID & Machine-ID assignment](#eos-17831-bootstrap-storage-set-id-and-machine-id-assignment)
-- [EOS-17235, EOS-16869: Destroy cortx on single node vm](#eos-17235-eos-16869-destroy-cortx-on-single-node-vm)
-- [EOS-18270: 3-node VM deployment implementation and integration Phase-2 3rd Party SW](#eos-18270-3-node-vm-deployment-implementation-and-integration-phase-2-3rd-party-sw)
-- [EOS-13889: Move generated configs directory in provsioner](#eos-13889-move-generated-configs-directory-in-provsioner)
-- [EOS-16883: SW Upgrade structure validation](#eos-16883-sw-upgrade-structure-validation)
-- [EOS-18732: SW upgrade bundle check-sum verification](#eos-18732-sw-upgrade-bundle-check-sum-verification)
-- [EOS-18743: RELEASE.INFO file exists and has the necessary scheme](#eos-18743-releaseinfo-file-exists-and-has-the-necessary-scheme)
+| Status (proposed/validated/needs attention) | Feature (JIRA/Description) | Test Suite | Comments (If needs attention, why? Any special note for the particular case, like specific environment treatment) |
```

Comment:
I believe better to use `Test Case` or `Test Scenario` insetad of `Test Suite` since we consider some somehow focused separated scenario instead of a suite (a set of something)

Reply:
Done